from fastapi import HTTPException


def user_create():
    ...
